[local development bug] yarn build generates /lib/index.js instead of /lib/src/index.js

just did a clean install and I get another error upon yarn start: Cannot find module './lib/src/index.js'. it seems Typescript is generating ./lib/index.js instead of ./lib/src/index.js. Not sure why it changed

Originally from telegram: by @whilefoo https://t.me/UbiquityDAO/31132/31145

@pavlovcik on a clean build, touching nothing
```typescript
$ probot run ./lib/src/index.js
/home/ubiquibot/node_modules/resolve/lib/sync.js:111
    var err = new Error("Cannot find module '" + x + "' from '" + parent + "'");
              ^
Error: Cannot find module './lib/src/index.js' from '/home/ubiquibot'
    at resolveSync (/home/ubiquibot/node_modules/resolve/lib/sync.js:111:15)
    at resolveAppFunction (/home/ubiquibot/node_modules/probot/lib/helpers/resolve-app-function.js:34:23)
    at combinedApps (/home/ubiquibot/node_modules/probot/lib/run.js:101:79)
    at async Server.load (/home/ubiquibot/node_modules/probot/lib/server/server.js:60:9)
    at async run (/home/ubiquibot/node_modules/probot/lib/run.js:105:9) {
  code: 'MODULE_NOT_FOUND'
}
```

PRing...

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "lint": "eslint --ext .ts ./src",
     "start:serverless": "tsx src/adapters/github/github-actions.ts",
     "start:watch": "nodemon --exec 'yarn start'",
-    "start": "probot run ./lib/src/index.js",
+    "start": "probot run ./lib/index.js",
     "prepare": "husky install"
   },
   "dependencies": {

